test/test_cabin_helper.rb

```diff
@@ -0,0 +1,199 @@
+$: << File.dirname(__FILE__)
+$: << File.join(File.dirname(__FILE__), "..", "lib")
+
+require "rubygems"
+require "minitest-patch"
+require "cabin"
+
+module RSpec
+  class Confi
+    def include(*)
+    end
+    def before(*)
+    end
+    def after(*)
+    end
+  end
+
+  def self.configure
+    yield Confi.new
+  end
+end
+
+require "cabin/rspec/cabin_helper"
+require "minitest/autorun" if __FILE__ == $0
+
+describe "RSpec helper for Cabin" do
+
+  class RSpecContext
+    include CabinHelper
+  end
+
+  before do
+    @context = RSpecContext.new
+    @receiver = RSpecContext::LoggingReceiver.new
+    @matcher = RSpecContext::ReceiveLogMessage.new(@receiver, "Foo", {})
+  end
+
+  test "context should have #receive_log_message" do
+    assert(@context.respond_to?(:receive_log_message))
+  end
+
+  test "context#receive_log_message should return a matcher" do
+    assert_equal(@context.receive_log_message("foo").class, @matcher.class)
+  end
+
+  test "context should have #log_receiver" do
+    assert(@context.respond_to?(:log_receiver))
+  end
+
+  test "context#log_receiver should return a LoggingReceiver instance" do
+    assert_equal(@context.log_receiver.class, @receiver.class)
+  end
+
+  test "context should have #new_subscribed_cabin_channel" do
+    assert(@context.respond_to?(:new_subscribed_cabin_channel))
+  end
+
+  test "context#new_subscribed_cabin_channel should return a Cabin Channel instance" do
+    assert_equal(@context.new_subscribed_cabin_channel(:info, @receiver).class.name, "Cabin::Channel")
+  end
+
+  describe RSpecContext::LoggingReceiver do
+    before do
+      @subject = RSpecContext::LoggingReceiver.new
```

Comment:
maybe you are correct. i looked at the source to see whether ```described_class``` was provided, seems not.

Before, in most of my situations, the subject was never as simple as ```SomeClass.new``` so i have a habitual blindness about implicit ```subject```